exporter/cortexexporter/DESIGN.md

```diff
@@ -0,0 +1,287 @@
+
+# **OpenTelemetry Collector Cortex Exporter Design**

```

Comment:
Should this be named "Prometheus Remote Write API Exporter"? Idea being if other product supports that can use the same exporter, it is not tied to Cortex, correct?

Reply:
We could make such a rename. Other products that are integrated with Prometheus Remote Write API should be supported in theory; however, we will not be conducting tests for backends other than Cortex. We could work on validating other backends are supported by the exporter with the community in the future.  

Also, having two exporters with the name "Prometheus" in it leads to confusion and questions like whether this exporter can be used as a push-based exporter for Prometheus. But we could add a explicit call-out in our readme to highlight the differences. What are your thoughts?  @alolita